setup.py

```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8 -*-
+
+import pathlib
+from setuptools import setup
+
+root_dir = pathlib.Path(__file__).parent
+
+def read(*names, **kwargs):
+    with open(root_dir.joinpath(*names), "r") as fh:
+        return fh.read()
+
+setup(
+    name="wikihow2zim",
+    version=read("wikihow2zim", "VERSION").strip(),
+    description="Make ZIM file from WikiHow articles",
+    long_description=read("README.md"),
+    long_description_content_type="text/markdown",
+    author=" ",
```

Comment:
Please fill-in those metadata. You can use `Kiwix Team` and `dev@kiwix.org` as well as `https://kiwix.org/`